tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/AmqpClientTestSupport.java

```diff
@@ -185,6 +187,7 @@ protected TransportConfiguration addAcceptorConfiguration(ActiveMQServer server,
       HashMap<String, Object> params = new HashMap<>();
       params.put(TransportConstants.PORT_PROP_NAME, String.valueOf(port));
       params.put(TransportConstants.PROTOCOLS_PROP_NAME, getConfiguredProtocols());
+      params.put(TransportConstants.NETTY_READ_TIMEOUT, readTimeout);
```

Comment:
3 ms seems very small even for test case, also test should test without setting this that error does occur. Eg assert the problem exists without setting.

Reply:
Actually it is 3 seconds. It is just typo in property string. I've tried to make test as you say. Check it in new commit. And thanks for reviewing! :)